sklearn/utils/tests/test_multiclass.py

```diff
@@ -289,6 +289,14 @@ def test_type_of_target():
                'representation. Sequence of sequences are no longer supported;'
                ' use a binary array or sparse matrix instead.')
         assert_raises_regex(ValueError, msg, type_of_target, example)
+        
+    try:
+        from pandas import SparseSeries
+    except ImportError:
+        pass
+    y = SparseSeries([1,0,0,1,0])
```

Comment:
So I'm seeing the error in the automatic checks that states: "UnboundLocalError: local variable 'SparseSeries' referenced before assignment"  Do I need to put all of the test code within the `try` block?